internal/config/types.go

```diff
@@ -20,11 +20,17 @@ type ControlServer struct {
 }
 
 // Metrics config.
+// Deprecated.
 type Metrics struct {
-	Enabled    bool   `yaml:"enabled" json:"enabled" env:"ENABLED"`
 	ClientType string `yaml:"client_type" json:"client_type" env:"CLIENT_TYPE" env-default:"noop"`
 }
 
+// PrometheusMetrics config.
+type PrometheusMetrics struct {
+	Enabled bool   `yaml:"enabled" json:"enabled" env:"ENABLED"`
+	Port    string `yaml:"port" json:"port" env:"PORT" env-default:":9090"`
```

Comment:
Thanks for doing this – That's cool with me, however, let's rename `Port` to `Address`, as it's a listen address, not just port.